Lombiq.Vsix.OrchardShared/Commands/InjectDependencyCommand.cs

```diff
@@ -1,108 +1,108 @@
-using Lombiq.Vsix.Orchard.Constants;

```

Comment:
Valami fura, csomó fájlra úgy mutat változást itt, hogy nem látok az átnevezésen kívül különbséget.

Reply:
Sok fájlt csak át kellett rakni a Shared project-be

Reply:
Ja, de itt valami egyéb változást is jelez. EOL tud ilyet okozni pl. (de azt megnéztem és nem úgy tűnik), meg encodingok.

Reply:
Ez továbbra is fennáll, és így nem lehet rendesen review-zni. Lásd: https://github.com/Lombiq/Orchard-Data-Tables/pull/80#discussion_r790869513

Reply:
Továbbra is volt több ilyen fájl, de mostanra már sok lett a várakozásból, szóval azokat inkább kimásoltam és úgy diffeltem.